packages/text-input-react/example/index.tsx

```diff
@@ -16,28 +16,44 @@ const TextFieldDemo = () => {
     return (
         <>
             <div className="side-by-side">
-                <TextField label="Fornavn" value={value} onChange={handleChange} placeholder={"Norsk"} />
-                <TextField label="Fornavn" value={"Per"} onChange={handleChange} readOnly />
+                <TextField
+                    label="Fornavn"
+                    value={value}
+                    onChange={handleChange}
+                    placeholder={"Norsk"}
+                    variant="small"
+                    helpLabel="La oss se..."
+                />
+                <TextField label="Fornavn" value={"Per"} onChange={handleChange} readOnly variant="small" />
             </div>
 
-            <TextField label="Telefon" type="tel" value={value} onChange={handleChange} />
+            <TextField label="Pelefon" type="tel" value={value} onChange={handleChange} placeholder="999 99 999" />
```

Comment:
Pelefon?

Reply:
Haha! We were testing visual alignment and wanted a left-aligned character. Forgot to change it back 😅 Either way, we need better and domain relevant examples of most components…